src/jdbc/cubrid/jdbc/jci/UResCache.java

```diff
@@ -34,22 +34,30 @@
 public class UResCache {
 	UBindKey key;
 
-	private boolean used;
+	private boolean inUse;
```

Comment:
naming; isInUsed

Reply:
I think "inUse" is better than your suggestion.

Reply:
ok.